docs/Upgrade.md

```diff
@@ -1665,6 +1665,15 @@ This also means you can now use [`yup`](https://github.com/jquense/yup), [`zod`]
 
 With `react-hook-form`, you can't have both form level validation and input level validation. This is because form level validation is meant to be used for [schema based validation](https://react-hook-form.com/api/useform#validationResolver).
 
+The default mode for validation is 'onSubmit', so if you want to have input level validation triggered before submission, you can try a different [validation strategy](https://react-hook-form.com/api/useform/) by passing `mode` prop to the form:
```

Comment:
I'd create a dedicated section for this one, before "Validation: Form Level & Input Level Are Mutually Exclusive", called "Input-Level Validation Now Triggers on Submit"